nevergrad/optimization/test_optimizerlib.py

```diff
@@ -134,7 +134,8 @@ def check_optimizer(
 ]
 
 
-UNSEEDABLE: tp.List[str] = []
+# Unfortunately, it looks like there is no seed for modcma.
+UNSEEDABLE: tp.List[str] = ["modcma"]
```

Comment:
```suggestion
UNSEEDABLE: tp.List[str] = ["modcma"]
```

That's concerning though, that seems weird